internal/layer2/announcer.go

```diff
@@ -209,21 +218,23 @@ func (a *Announce) SetBalancer(name string, ip net.IP) {
 }
 
 // DeleteBalancer deletes an address from the set of addresses we should announce.
-func (a *Announce) DeleteBalancer(name string) {
+//
+// Return the IP if it is no longer in use.
```

Comment:
```suggestion
// Returns the IP if it is no longer in use.
```

I think the normal tense for doc comments is simple present rather than imperative.